scripts/generate_ws_config.sh

```diff
@@ -227,6 +240,10 @@ client_secret=${local_DATAFED_GLOBUS_APP_SECRET}
 # This is the address to talk with the core server which is listening on 
 # port 7513, assuming internal network.
 server_address=tcp://${local_DATAFED_CORE_ADDRESS_PORT_INTERNAL}
+
+[operations]
+# This is the tag associated with a Google Analytics installation that metrics will be sent to.
+google_analytics_tag=${local_DATAFED_GOOGLE_ANALYTICS_TAG}
```

Comment:
I like that this is in an operations section, why did you pick operations instead web?

Reply:
Well that entire file is configuration for the web server so I thought a new operations category was more descriptive to what this configuration option was for.